projects/index.html

```diff
@@ -1,11 +1,11 @@
 ---
-title: Projects
-excerpt: "This is where we keep a collection of some of the projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
+title: Project Showcase
+excerpt: "This is where we keep a collection of some of the most interesting projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
```

Comment:
Maybe add a project or two that could be reproduced by a beginner (like demos from one of the libraries), and also introduce classification of the projects by level (beginner, intermediate, advanced)?